src/social-network/defaults/shouldDisplayWelcome.ts

```diff
@@ -1,16 +1,17 @@
 import Services from '../../extension/service'
 import { getStorage } from '../../utils/browser.storage'
-import { SocialNetworkUI } from '../ui'
+import { getActivatedUI } from '../ui'
 
-export async function shouldDisplayWelcomeDefault(this: SocialNetworkUI) {
-    if (typeof this.networkIdentifier === 'function')
+export async function shouldDisplayWelcomeDefault() {
+    const netId = getActivatedUI().networkIdentifier
+    if (typeof netId === 'function')
```

Comment:
Just remove this case since the def has changed

Reply:
e84d6b4b4c013720e40b731227e71649b7afc48e